modules/OFConnectionManager2/utest/main.c

```diff
@@ -1093,6 +1096,43 @@ subbundle_finishes[] = {
     subbundle_finish,
 };
 
+uint32_t invoke_pre_start[NUM_SUBBUNDLES];
+uint32_t invoke_post_finish[NUM_SUBBUNDLES];
```

Comment:
even though this is just the utest, suggest renaming "pre_start" to "start3" and "post_finish" to "finish3" throughout the utest.  "pre_start" implies one step before "start", and "post_finish" implies one step after "finish", whereas i think we just want names that don't overlap with the existing names from testing set2